src/styles/_base.sass

```diff
@@ -143,3 +143,10 @@ div.tooltip-inner
 	padding: 0px 4px
 	line-height: 1.25em
 	border-radius: 2px
+
+.padding-left-10
```

Comment:
I'm not in favour of these type of class names, as it means we're moving layout/presentation into the DOM, which we should try to avoid. 

 Looks like padding-left-20 and padding-left-30 are also not used in this case.

Reply:
You're right, changed.